src/include/homestore/blkdata_service.hpp

```diff
@@ -142,7 +142,7 @@ class BlkDataService {
      *
      * @param bid The MultiBlkId of the block to commit.
      */
-    void commit_blk(MultiBlkId const& bid);
```

Comment:
Isn't it better that the API doesn't care about ignore failure or not and let caller handle that and commit_blk simply returns error or not?

Reply:
yes, actually ， I wanted to do this also. 
my concern here is that will change the API statement of blkservice , so according to semver ,  we need to bump a MAJOR version of homestore. 
so , i want to do this in a separate PR.